x-pack/plugins/security_solution/public/cases/components/tag_list/index.tsx

```diff
@@ -99,15 +99,15 @@ export const TagList = React.memo(
         <EuiHorizontalRule margin="xs" />
         <MyFlexGroup gutterSize="xs" data-test-subj="case-tags">
           {tags.length === 0 && !isEditTags && <p data-test-subj="no-tags">{i18n.NO_TAGS}</p>}
-          <EuiBadgeGroup>
-            {tags.length > 0 &&
-              !isEditTags &&
-              tags.map((tag) => (
+          {tags.length > 0 && !isEditTags && (
```

Comment:
it looks like that we have the same piece of code with the same logic in the next file `x-pack/plugins/security_solution/public/cases/components/user_action_tree/helpers.tsx` Line 58. It might be a good idea to have a generic component for it.